src/utils/achievements.ts

```diff
@@ -31,7 +31,7 @@ export const getAchievementDescription = (campaign: Campaign): TranslatableText
   switch (campaign.type) {
     case 'ifo':
       return {
-        key: 'Committed more than $5 worth of LP in the %title% IFO',
+        key: 'Committed more than $5 worth in the %title% IFO',
```

Comment:
'Committed more than $%amount% %token% in the %title% IFO'

amount: Lowest price, IFO: $100, cIFO: $10
token: Subscription tokens used by IFO.

Reply:
Please change the wording to
`Committed more than $5 in the %title% IFO`